architecture/topics/auth_enabled_registry.adoc

```diff
@@ -0,0 +1,114 @@
+// This assembly is included in the following assemblies:
+//
+// <architecture/infrastructure_components/image_registry#auth-enabled-registry.adoc>
+//<install_config/registry/index#auth-enabled-registry.adoc>
+
+// This assembly can be included from other assemblies using the following
+// include statement:
+
+// The following line is necessary to allow assemblies be included in other
+// assemblies. It restores the `context` variable to its previous state.
+:parent-context-of-creating-build-inputs: {context}
+
+// The file name and the ID are based on the assembly title.
+// For example:
+// * file name: assembly_my-assembly-a.adoc
+// * ID: [id='assembly_my-assembly-a_{context}']
+// * Title: = My assembly A
+
+//Concept
+All container images available through the Red Hat Container Catalog are hosted
+on an image registry, `registry.access.redhat.com`. The Red Hat Container Catalog
```

Comment:
There are several double spaces in this PR. Please check.

I'd say "With {product-title} version 3.11, the Red Hat Container Catalog moved from..."